.gitlab/ci/on-push.yml

```diff
@@ -46,37 +63,46 @@ create-instances:
     - rm -rf $ARTIFACTS_FOLDER/uploadable_packs
     - python3 ./Tests/scripts/update_conf_json.py
     - cp "./Tests/conf.json" "$ARTIFACTS_FOLDER/conf.json"
+    - section_end "Create Content Artifacts and Update Conf"
+    - section_start "Collect Tests and Content Packs"
     - |
       if [ -n "${INSTANCE_TESTS}" ]; then
         echo "Skipping - not running in INSTANCE_TESTS build"
       else
-        echo "====== Collecting tests and content packs ======"
         [ -n "${NIGHTLY}" ] && IS_NIGHTLY=true || IS_NIGHTLY=false
         python3 ./Tests/scripts/collect_tests_and_content_packs.py -n $IS_NIGHTLY
       fi
+    - section_end "Collect Tests and Content Packs"
+    - section_start "Calculate Packs Dependencies"
     - python3 ./Tests/Marketplace/packs_dependencies.py -i ./Tests/id_set.json -o $ARTIFACTS_FOLDER/packs_dependencies.json
+    - section_end "Calculate Packs Dependencies"
+    - section_start "Prepare Content Packs for Testing"
     - |
       if [[ -n "${CONTRIB_BRANCH}" ]]; then
-        echo "Skipping Preparing Content Packs For Testing since nightly instances uses production bucket"
+        echo "Skipping Preparing Content Packs For Testing since CONTRIB_BRANCH was $CONTRIB_BRANCH"
       else
-        echo "====== Preparing Content Packs For Testing ======"
         ./Tests/scripts/prepare_content_packs_for_testing.sh "$GCS_MARKET_BUCKET"
       fi
+    - section_end "Prepare Content Packs for Testing"
+    - section_start "Zip Marketplace Packs"
     - |
       if [[ $CI_COMMIT_BRANCH != master ]] && [[ $CI_COMMIT_BRANCH != 20\.* ]] && [[ $CI_COMMIT_BRANCH != 21\.* ]]; then
-        echo "Skipping packs download to artifact on non master or release branch"
+        echo "Skipping packs download to artifact on branch that is not master or a release branch"
+        echo "CI_COMMIT_BRANCH=$CI_COMMIT_BRANCH"
       else
         ZIP_FOLDER=$(mktemp -d)
         python3 ./Tests/Marketplace/zip_packs.py -b 'marketplace-dist' -z $ZIP_FOLDER -a $ARTIFACTS_FOLDER -s $GCS_MARKET_KEY
       fi
+    - section_end "Zip Marketplace Packs"
+    - section_start "Create Instances"
     - |
-      echo "====== Creating instances ======"
       [ -n "${TIME_TO_LIVE}" ] && TTL=${TIME_TO_LIVE} || TTL=300
       python3 ./Tests/scripts/awsinstancetool/aws_instance_tool.py -envType "$IFRA_ENV_TYPE" -timetolive $TTL -outfile "$ARTIFACTS_FOLDER/env_results.json"
-  after_script:
```

Comment:
will this now inherit the default after_script?

Reply:
There is no default `after_script` anymore.